examples/sha256/table16.rs

```diff
@@ -158,82 +179,79 @@ impl<F: FieldExt> Table16Chip<F> {
             ],
         );
 
-        let compression = Compression::configure(
+        let compression = CompressionConfig::configure(
             meta,
             lookup_inputs.clone(),
             message_schedule,
             extras,
             perm.clone(),
         );
 
-        let message_schedule =
-            MessageSchedule::configure(meta, lookup_inputs, message_schedule, extras, perm);
+        let message_schedule = MessageScheduleConfig::configure(
+            meta,
+            lookup_inputs.clone(),
```

Comment:
I don't believe this clone is necessary:
```suggestion
            lookup_inputs,
```